docs/source/contribution/technical_steering_committee.md

```diff
@@ -2,21 +2,24 @@
 
 Kedro is an incubating project within [LF AI & Data](https://lfaidata.foundation/).
 
-The term "Technical Steering Committee" (TSC) describes the group of Kedro maintainers. We list Kedro's [current](#current-maintainers) and [past](#past-maintainers) maintainers on this page.
+The term "Technical Steering Committee" (TSC) describes the group of Kedro maintainers and committers ("contributors"). We list Kedro's [current](#current-maintainers) and [past](#past-maintainers) maintainers on this page.
 
 The TSC is responsible for the project's future development; you can read about our duties in our [Technical Charter](https://github.com/kedro-org/kedro/blob/main/kedro_technical_charter.pdf). We accept new members into the TSC to fuel Kedro's continued development.
 
 On this page we describe:
 
 - [Responsibilities of a maintainer](#responsibilities-of-a-maintainer)
 - [Requirements to become a maintainer](#requirements-to-become-a-maintainer)
+- [Responsibilities of a committer](#responsibilities-of-a-committer)
 - [Current maintainers](#current-maintainers)
 - [Past maintainers](#past-maintainers)
-- [Application process](#application-process)
+- [Application process for maintainers](#application-process-for-maintainers)
 - [Voting process](#voting-process)
 
 ## Responsibilities of a maintainer
 
+Maintainers are the voting members of the TSC.
```

Comment:
```suggestion
Maintainers are the voting members of the TSC. Beyond contributing to regular development, they are responsible for longer-term and strategic decisions.
```
Maybe there's a better way to word this added responsibility.

Reply:
Since a list of responsibilities follows, I'll adjust this wording to make it clear that it's a summary of sorts.